### DISCUSSION TITLE
Search commits but remove results from forks?

### DISCUSSION THREAD

**Discussion by HaraldNordgren** [community/community#22876] [New to GitHub]
<p>I’m using <a href="https://help.github.com/articles/searching-commits/" rel="nofollow noopener">https://help.github.com/articles/searching-commits/</a> to try get generate a nice overview of all my open-source contributions.</p>
<p>But because of _forking_, many of the commits are duplicated on multiple repos. So they show up in the results multiple times. Example:</p>
<p><a href="https://github.com/search?o=desc&amp;p=3&amp;q=author-email%3Aharaldnordgren%40gmail.com+is%3Apublic+-user%3AHaraldNordgren+-user%3Agifflarn+-user%3Aeveryonehelp+-user%3Adatateknik-lth+-user%3ACantemo+-user%3Abetalo-sweden&amp;s=committer-date&amp;type=Commits" rel="nofollow noopener">https://github.com/search?o=desc&amp;p=3&amp;q=author-email%3Aharaldnordgren%40gmail.com+is%3Apublic±user%3AHaraldNordgren±user%3Agifflarn±user%3Aeveryonehelp±user%3Adatateknik-lth±user%3ACantemo±user%3Abetalo-sweden&amp;s=committer-date&amp;type=Commits</a></p>
<p>Is it possible to filter commits by applying a uniqueness constraint on the commit hash. Or an option to remove forked repos?</p>

**Comment by ghost** [comment, marked as answer]
<p>Hey there <a class="mention" href="/haraldnordgren">@haraldnordgren</a>, help’s arrived.</p>
<p>Yikes, that doesn’t sound right. GitHub is meant to ignore forks from search results by default, so it might be worth contacting the folks at GitHub know this is happening. You can get in touch with them here: <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a>.</p>
<p>In the meantime, you can always use the  <strong>fork:false</strong>  flag to exclude forked results in search :slight_smile: There’s also some more info on searching here: <a href="https://help.github.com/articles/searching-code/" rel="nofollow noopener">https://help.github.com/articles/searching-code/</a></p>
<p>Let me know if this works for you!</p>
